style.css

```diff
@@ -1,582 +1,714 @@
+  top: 0;
+  width: 100%;
+  padding: 1rem 0;
   display: flex;
-  gap: 20px;
-  margin-top: 12px;
-}
-
-section {
-  margin: 128px 10% 136px 10%;
-}
-
-.top {
-  padding-bottom: 138px;
+  align-items: center;
+  justify-content: space-between;
 }
 
-.logo {
+header .my-logo {
   margin-left: 24px;
-  font-family: "poppins", sans-serif;
+  color: #6070ff;
+  font-family: "Poppins", sans-serif;
   font-weight: bold;
   font-size: 18px;
-  color: #6070ff;
   line-height: 20px;
 }
 
-.fas {
-  margin-right: 24px;
-  color: #acb7c3;
+section ul {
+  margin-top: 12px;
+  display: flex;
+  gap: 20px;
 }
 
-.hamburger {
-  margin-top: 58px;
-  margin-right: 24px;
-  background-color: white;
-  border: 0;
+header nav ul {
   display: none;
 }
 
-header .nav-list {
-  display: flex;
+.fa-bars {
+  margin-right: 24px;
+  color: #6070ff;
 }
 
-header .nav-list .close-menu {
-  display: none;
+/* ------- Section ------- */
+.section-top {
+  margin: 0 24px 136px 24px;
+  padding-top: 128px;
 }
 
-header .nav-list li {
-  margin-right: 32px;
+.section-title {
+  color: #172b4d;
+  font-family: "Poppins", sans-serif;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  word-wrap: break-word;
 }
 
-.hamburger i {
-  color: #6070ff;
+section p {
+  margin-top: 12px;
+  color: #344563;
+  font-family: "Poppins", sans-serif;
+  font-size: 16px;
+  line-height: 24px;
 }
 
-.project {
-  /* margin: 114px 24px 114px 24px; */
+section .main-text {
+  color: #7f8cff;
+  margin-top: 12px;
+}
+
+/* ------- Section - Cards Project ------- */
+.cards-project {
+  padding-top: 114px;
+}
+
+.cards-project-list {
+  width: 88%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 114px;
   display: grid;
   gap: 88px;
 }
 
-.project-card {
+.card {
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
   box-sizing: border-box;
   border-radius: 16px;
   padding: 16px;
-  border: 1px solid #dfe1e6;
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  background-color: #fff;
 }
 
-.pix {
-  height: 220px;
+.card img:not(:last-child) {
   width: 100%;
-  display: unset;
+  margin-top: 12px;
+  height: auto;
 }
 
-.project-title {
-  font-family: "poppins", sans-serif;
+.card .card-title {
+  color: #172b4d;
+  font-family: "Poppins", sans-serif;
   font-weight: bold;
   font-size: 32px;
   line-height: 44px;
-  color: #172b4d;
   margin-top: 12px;
 }
 
-.project-description {
+.card .card-description {
+  color: #344563;
+  font-family: "Poppins", sans-serif;
   font-weight: 600;
   font-size: 13px;
   line-height: 16px;
-  color: #344563;
   margin-top: 20px;
-  font-family: "poppins", sans-serif;
 }
 
-span {
+.card span {
   color: #7a869a;
-  font-family: "poppins", sans-serif;
+  font-family: "Poppins", sans-serif;
   font-weight: 600;
   font-size: 13px;
   line-height: 16px;
 }
 
-.project-content {
+.card .card-text-content {
+  color: #344563;
+  font-family: "Poppins", sans-serif;
   font-size: 15px;
   line-height: 24px;
```

Comment:
- [ ] This section overflowing its parent container on mobile resolution. (Using static height for the popup card causes this issue.)
Please be sure you didn't break the functionality of sections when you are updating them. 

![Screenshot from 2021-06-24 23-02-02](https://user-images.githubusercontent.com/1642014/123325871-398c0400-d541-11eb-95ff-6329520a6dbb.png)